### DISCUSSION TITLE
Disable LazyExtra R Lang for .qmd (Quarto) files

### DISCUSSION THREAD

**Discussion by weberam2** [LazyVim/LazyVim#4572] [General]
Hello,

Is there a way for me to have the R Lang Extra installed, but prevent it from working with .qmd / quarto files?

I tried:

```
return {
  "R-nvim/R.nvim",
  ft = { "r", "rnoweb", "rhelp", "rmd" },
```

but that didn't work...

**Comment by dpetka2001** [comment]
https://github.com/R-nvim/R.nvim/blob/ad7035e574677fa9bc339aaf7b38c317ffc0fd3b/doc/R.nvim.txt#L1796-L1802

**Reply by weberam2** [reply]
YES THANK YOU!

This worked:

```
return {
  "R-nvim/R.nvim",
  config = function()
    vim.g.R_filetypes = { "r", "rmd", "rnoweb", "rhelp" }
  end,
```
